src/ixdat/readers/reading_tools.py

```diff
@@ -11,6 +11,17 @@
 STANDARD_TIMESTAMP_FORM = "%d/%m/%Y %H:%M:%S"  # like '31/12/2020 23:59:59'
 USA_TIMESTAMP_FORM = "%m/%d/%Y %H:%M:%S"  # like '12/31/2020 23:59:59'
 FLOAT_MATCH = "[-]?\\d+[\\.]?\\d*(?:e[-]?\\d+)?"  # matches floats like '5' or '-2.3e5'
+DEFAULT_READER_NAMES = {
+    ".mpt": "biologic",
+    ".tsv": "zilien",
+    ".xrdml": "xrdml",
+    ".avg": "avantage",
+}
```

Comment:
Two good points here. I've made these changes:
- Now if a suffix is provided without a "." at the start, the "." is added to match the syntax of `pathlib.Path.suffix`. So the user gets the same result when calling with `suffix="mpt"` as they do with `suffix=".mpt"`
- `Measurement.read_set` now raises a `ReadError` asking the user to check their inputs and/or files in case of an empty file list, rather than returning `None`.

Reply:
Great!